playbooks/roles/mailing-list/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "restart mailman3"
+  service:
+    name: "{{ item }}"
+    state: restarted
+  with_items:
+    - "mailman3"
+    - "mailman3-web"
```

Comment:
@swalladge This should really be two separate handlers (`notify` takes a list). And you don't need all the quotes.